### ISSUE TITLE
Annotation Check on Custom dataset 

### ISSUE THREAD

**Issue by abhigoku10** [dbolya/yolact#135]
@dbolya  i have around 80K custom dataset of additional classes(60) which i have combined with 1.2L Mscoco classes (80) when i put it for training i am facing few issues .

1. The training gets stuck for 6350,13050,3000 iterations of first epoch .
2. I get "Memory error"  when the training starts 

THe above mentioned error is uncertain and not been to  replicate  . Earlier teh trainings were happening smoothly . Any idea on debugging the issues , either with annotations or mask generated for the huge dataset

**Comment by dbolya** [comment]
For the memory error (2), that would make sense since you added more classes and so there are more parameters. You could probably just reduce the batch size to 7 and increase the training time (no need to disable batch norm either).

On the other hand, I don't know what you mean by (1). Can you elaborate?

**Comment by abhigoku10** [comment]
1. the training stops at some of the iterations no error or warning , it will remain the same for long time without going to next iteration

**Comment by dbolya** [comment]
That's very weird--I've never experienced that. Are you sure it's not just computing the validation mAP?

**Comment by abhigoku10** [comment]
@dbolya  no its not computing validation mAP since if it was then on the terminal it will print "Calculating " text which was not shown , any pointers to debug this

**Comment by dbolya** [comment]
Hmm when it finally does go to the next iteration, does it show a very large ETA, or does it just skip some iterations (trying to check whether it's actually freezing or just not printing for a while)?

A couple things you can do to debug: save right before one of those iterations, load the weights and see if it happens again; add more print statements to see where's the last place it got to before freezing; watch nvidia-smi while it freezes to see if maybe the GPU is being used by something else (or does something else that's weird).

**Comment by abhigoku10** [comment]
sure i shall add more print statements , i checked up with nvidia-smi during the start of the training the GPU usage is 75-80% when i the training gets stucked it reduces to 6%

**Comment by dbolya** [comment]
You might also want to check `top` too, since if the GPU usage is reduced to 6% there might be something happening on your CPU.

**Comment by abhigoku10** [comment]
@dbolya i found a strange behaviour i was training resnet50 from scratch which i have never done , might be this causing issue

**Comment by dbolya** [comment]
Hmm yeah I haven't tried training from scratch

**Comment by abhigoku10** [comment]
@dbolya as usuall thanks for the support i shall debug more and get back to u on it
